cumulus_library/study_parser.py

```diff
@@ -307,18 +373,85 @@ def run_table_builder(
         for file in self.get_table_builder_file_list():
             self._load_and_execute_builder(file, cursor, schema, verbose)
 
-    def run_counts_builder(
+    def run_counts_builders(
         self, cursor: DatabaseCursor, schema: str, verbose: bool = False
     ) -> None:
         """Loads counts modules from a manifest and executes code via BaseTableBuilder
 
+        While a count is a form of statistics, it is treated separately from other
+        statistics because it is, by design, always going to be static against a
+        given dataset, where other statistical methods may use sampling techniques
+        or adjustable input parameters that may need to be preserved for later review.
+
         :param cursor: A PEP-249 compatible cursor object
         :param schema: The name of the schema to write tables to
         :param verbose: toggle from progress bar to query output
         """
         for file in self.get_counts_builder_file_list():
             self._load_and_execute_builder(file, cursor, schema, verbose)
 
+    def run_statistics_builders(
+        self,
+        cursor: DatabaseCursor,
+        schema: str,
+        verbose: bool = False,
+        stats_build: bool = False,
+    ) -> None:
+        """Loads statistics modules from toml definitions and executes
+
+        :param cursor: A PEP-249 compatible cursor object
+        :param schema: The name of the schema to write tables to
+        :keyword verbose: toggle from progress bar to query output
+        :keyword stats_build: If true, will run statistical sampling & table generation
+        """
+        if not stats_build:
+            return
+        for file in self.get_statistics_file_list():
+            # This open is a bit redundant with the open inside of the PSM builder,
+            # but we're letting it slide so that builders function similarly
+            # across the board
+            iso_timestamp = datetime.now().replace(microsecond=0).isoformat()
```

Comment:
Same naive/aware comment here. I wish python had never allowed naive times. A scourge upon us all. Might want to make a little utility method to help keep you honest next time you need a timestamp.

Reply:
just add it to the list of 'things we should enforce check because i do them all the time' along with debugging statements getting missed.